glean-core/rlb/tests/schema.rs

```diff
@@ -115,7 +115,7 @@ fn validate_against_schema() {
         Ok(()) => {}
         Err(e) => {
             let errors = e.map(|e| format!("{}", e)).collect::<Vec<_>>();
-            assert!(false, "Data: {:#?}\nErrors: {:#?}", data, errors);
+            unreachable!("Data: {:#?}\nErrors: {:#?}", data, errors);
```

Comment:
Shouldn't this be more like `panic!` to communicate intent better? This code is clearly not _unreachable_, but given that it is a test we want it to blow up (`unreachable!` will also just result in a panic, but the error message will include "internal error: entered unreachable code")

Reply:
Sure -- no problem making this `panic!` instead.